src/Sarif.Viewer.VisualStudio/CodeLocationObject.cs

```diff
@@ -48,11 +48,28 @@ public Region Region
                 if (value != _region)
                 {
                     _region = value;
+
+                    if (LineMarker != null)
+                    {
+                        LineMarker.Region = _region;
+                    }
+
                     NotifyPropertyChanged("Region");
+                    NotifyPropertyChanged("RegionDisplayString");
                 }
             }
         }
 
+        public string RegionDisplayString
```

Comment:
>RegionDisplayString [](start = 22, length = 19)

This property (renamed from LocationDisplayString) had to be moved up to this base class so the Region setter above could call NotifyPropertyChange for this property. This causes the region information shown after the source file name in the Locations view to update when the region is updated, e.g. when the user locates a file and we use the FileRegionsCache to flesh out the region properties.